Externals/libtheora_static.vcxproj

```diff
@@ -95,4 +95,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
A new line on the end of file was added, please, remove this change

Reply:
@Xottab-DUTY fixed